.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  tox:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version:
+        - 3.6
+        - 3.7
+        - 3.8
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install tox
+      run: |
+        python -m pip install --upgrade tox
+    - name: Initialize tox envs
+      run: |
+        python -m tox --parallel auto --notest
+    - name: Test with tox
+      run: |
+        python -m tox --parallel 0
```

Comment:
Whats the advantage over calling tox as module via the python executable, vs. the tox binary directly?

Reply:
I think it was to ensure the installed version was found and used (I copied it). I don't think it is required and is less idiomatic, so now I've removed.